docs/marketplace/user/merchant-portal-user-guides/202108.0/offers/managing-product-offers.md

```diff
@@ -16,108 +16,60 @@ Review the reference information in this article before you start, or just look
 
 To create a product offer:
 
-1. Click **Add Offer**.
-2. From the list of concrete products, select the necessary product you want to create an offer for, hover over the three dots, or just click the line and then click **Create Offer**.
-3. In the drawer, fill the necessary fields and click **Create**.
+1. Click **+Add Offer**.
+2. On the _Create Offer_ page, in the list of concrete products, click on the necessary product you want to create an offer for, or hover over the three dots and then click **Create Offer**. The drawer opens.
+3. In the _Offer status_ pane, select the **Offers is Active** check box to make the product offer visible on the Storefront. Otherwise, skip this step.
+4. In the _Merchant SKU_ pane, enter an SKU.
+5. In the _Stores_ pane, from the drop-down list, select stores for which the offer is available.
+6. In the _Available Stock_ pane, set configure the product offer quantity:
+    1. In the **QUANTITY** field, enter the quantity of the product currently available in stock.
+    2. (Optional) Select **Always in Stock** to make the product offer always available in terms of stock. Otherwise, when the number of orders exceeds the number of products set in the **QUANTITY** field, the product offer becomes out-of-stock on the Storefront.
+7. In the _Price_ pane, click **+Add** to set the price for your offer per store. The empty fields appear in the table below.
+   1. From the **Store** drop-down list, select the store for which the price is created.
+   1. From the **Currency** drop-down list, select the currency in which the price is defined.
+   1. In the **NET DEFAULT** field, enter a default net price. Use the period(.) separator for decimal values.
```

Comment:
Consider "User the period as a separator for decimal values". No need to indicate in parenthesis the period sign. Also please change in the sentences below.